js/foundation.drilldown.js

```diff
@@ -38,8 +38,6 @@ class Drilldown extends Plugin {
       'ARROW_DOWN': 'down',
       'ARROW_LEFT': 'previous',
       'ESCAPE': 'close',
-      'TAB': 'down',
-      'SHIFT_TAB': 'up'
```

Comment:
So how can `up` be triggered then?

Reply:
`SHIFT+TAB` is default keyboard behavior as well and will work without the need to define it in the component. You can see this PR in action at utulsa.edu